conf/iCubFindDependencies.cmake

```diff
@@ -72,8 +72,8 @@ checkandset_dependency(Qt5)
 
 if(icub_firmware_shared_FOUND AND ICUB_USE_icub_firmware_shared)
   # icub-firmware-shared 4.0.7 was actually a wrong version exported by icub-firmware-shared <= 1.15
-  if(icub_firmware_shared_VERSION VERSION_GREATER 4 OR icub_firmware_shared_VERSION VERSION_LESS 1.26.0)
-    message(FATAL_ERROR "An old version of icub-firmware-shared has been detected, but at least 1.26.0 is required")
+  if(icub_firmware_shared_VERSION VERSION_GREATER 4 OR icub_firmware_shared_VERSION VERSION_LESS 1.26.1)
+    message(FATAL_ERROR "An old version of icub-firmware-shared has been detected, but at least 1.26.1 is required")
```

Comment:
Hi @triccyx 

Why are we supposed to need this update?
If it's not strictly required, I would remove this change.

Reply:
I've seen that your commits contain @davidetome's one.
So, I'm afraid that you forked from devel to merge then in master.

Reply:
I've force pushed the removal of the unwanted commit.
@triccyx, please check it again.